reporter/src/main/kotlin/reporters/ReportTableModel.kt

```diff
@@ -87,10 +88,17 @@ data class ReportTableModel(
             val fullDefinitionFilePath: String,
 
             /**
-             * Information about if and why the project is excluded.
+             * Information about if and why the project is excluded by a [ProjectExclude].
              */
-            val exclude: ProjectExclude? = null
-    )
+            val projectExclude: ProjectExclude?,
```

Comment:
Why is the default value gone now?

Reply:
It was never required, the class is only instantiated in a single place which always provides a value for this parameter.